README.md

```diff
@@ -52,25 +62,29 @@ cd one-app
 npm ci --no-optional
 ```
 
-**Build a Module:**
+**Serve your Module to One App**
+
+At the root of your `one-app` repo, run:
 
 ```bash
-cd prod-sample/sample-modules/frank-lloyd-root/0.0.0
-npm ci
-cd ../../../..
-npm run serve-module prod-sample/sample-modules/frank-lloyd-root/0.0.0
-# this is used by the frank-lloyd-root module
-export ONE_CONFIG_ENV=development
+npm run serve-module <local-path-to-generated-module>
+# e.g. npm run serve-module ../my-first-module
```

Comment:
I like this having an example comment 👍

Reply:
I'm glad! I struggled with this with the previous docs. _Do I include the carrot brackets? What do they mean a local path?_ I think having a specific example of exactly what to type in the terminal minimizes room for error and frustration.